From f4e62af000116de93f0c9d1c5f8aaf40934d5a01 Mon Sep 17 00:00:00 2001
From: Paul Menzel <paulepanter@users.sourceforge.net>
Date: Fri, 15 May 2015 17:35:00 +0200
Subject: [PATCH 417/506] cb_timestamps.c: Add new time stamp descriptions
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add the descriptions of the “core”, that means no vendorcode or payload,
coreboot time stamps added up to coreboot commit a7d92441 (timestamps:
You can never have enough of them!) [1].

Running `coreboot_boottime` in the GRUB command line interface now shows
descriptions for all time stamps again on the ASRock E350M1.

[1] http://review.coreboot.org/9608
---
 grub-core/commands/i386/coreboot/cb_timestamps.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/grub-core/commands/i386/coreboot/cb_timestamps.c b/grub-core/commands/i386/coreboot/cb_timestamps.c
index 07fbb02..e72f38d 100644
--- a/grub-core/commands/i386/coreboot/cb_timestamps.c
+++ b/grub-core/commands/i386/coreboot/cb_timestamps.c
@@ -39,9 +39,17 @@ static const char *descs[] = {
   [2] = "before RAM init",
   [3] = "after RAM init",
   [4] = "end of romstage",
+  [5] = "start of verified boot",
+  [6] = "end of verified boot",
   [8] = "start of RAM copy",
   [9] = "end of RAM copy",
   [10] = "start of ramstage",
+  [11] = "start of bootblock",
+  [12] = "end of bootblock",
+  [13] = "starting to load romstage",
+  [14] = "finished loading romstage",
+  [15] = "starting LZMA decompress (ignore for x86)",
+  [16] = "finished LZMA decompress (ignore for x86)",
   [30] = "device enumerate",
   [40] = "device configure",
   [50] = "device enable",
-- 
2.4.3

